hack/config.yml

```diff
@@ -0,0 +1,35 @@
+global:
```

Comment:
Should we remove `hack` dir? you can put `.gitignore` to not accidently commit those (:

Reply:
Ah yes, removed :)